object-filter/src/main/java/org/infinispan/objectfilter/impl/syntax/JPAString.java

```diff
@@ -0,0 +1,10 @@
+package org.infinispan.objectfilter.impl.syntax;
+
+/**
+ * @author anistor@redhat.com
+ * @since 9.0
+ */
+interface JPAString {
+
+   String toJpaString();
```

Comment:
I noticed very similar implementations for ```toJpaString``` and ```toString``` across the *Expr classes, why this extra interface is needed?